src/components/Sushi.js

```diff
@@ -1,23 +1,34 @@
-import React from "react";
+import React, {useState} from "react"
+
+function Sushi({sushi, onSushiDelete, setPrice}) {
+  const [isEaten, setIsEaten] = useState(false)
+
+  const handleDeleteClick = () => {
+    const eatenSushi = sushi
+    setPrice(eatenSushi.price)
+    setIsEaten(true)
+
+    fetch(`http://localhost:3001/sushis/${sushi.id}`, {
+      method: "DELETE",
+    })
```

Comment:
Ah, interesting. I didn't think to delete the sushi from the DB, I guess it could work, but since the instructions didn't ask for it to persist, this was probably the more challenging way to do it. All good though!